shacl12-shacl-shacl/index.html

```diff
@@ -354,32 +356,39 @@
 
 		<section id="introduction">
 			<h2>Introduction</h2>
-			<p>TODO</p>
+			<p class="todo">TODO</p>
 		</section>
 
 		<section id="shapes">
 			<h2>Shapes</h2>
-			<p>TODO</p>
+			<p class="todo">TODO: This section will describe the Shapes created for SHACL-SHACL.</p>
+			<p>
+				These shapes are available in the RDF resource at <a href="http://www.w3.org/ns/shacl-shacl">http://www.w3.org/ns/shacl-shacl</a>.
```

Comment:
@caribouW3 , I could use a reminder.  Does putting SHACL-SHACL under `/ns` have different implications on its ability to be edited outside of a chartered working group's duration?

Reply:
The same conditions apply whether it's /ns/@@@ or a dated URI /YYYY/mm/@@@
If we make substantial non-backwards compatible changes, we should do that in a different namespace (e.g. 
/ns/shacl/12/@@@)
More explanations in https://www.w3.org/guide/editor/namespaces.html#Policy